synfig-studio/src/gui/app.cpp

```diff
@@ -1348,13 +1346,39 @@ App::get_default_accel_map()
 }
 
 /* === M E T H O D S ======================================================= */
+Glib::RefPtr<App> App::instance()
+{
+	static Glib::RefPtr<studio::App> app_reference = Glib::RefPtr<App>(new App());
+	return app_reference;
+}
+
+void App::run(int& argc, char**& argv)
```

Comment:
No need to pass references here. Previously, they were used due to an incorrect algorithm.

Reply:
Oh I read the method signature wrongly

Reply:
Done